cli.js

```diff
@@ -1,182 +1,88 @@
-          .option('depends', { describe: 'Execute steps for all components from depends list too', default: false, type: 'boolean' })
-          .demandOption(['steps'], 'Please provide steps(s) you need to run')
-      },
-      async (argv) => {
-        const {/*l,*/ f, /*t,*/ a} = await scope(argv.verbose, argv.detachPresets);
-        for (const component of a.resolve(argv.components)) {
-          const cntx = context.create(argv, utils.parseEnv(argv.env), argv.envFile, argv.save, argv.validate);
-          const steps = argv.steps.split(':');
-          if (argv.parallel) {
-            if (argv.depends) {
-              component.unfold(steps, f, argv.recursive, cntx);
-            } else {
-              component.run(steps, f, argv.recursive, cntx);
-            }
-          } else {
-            if (argv.depends) {
-              await component.unfold(steps, f, argv.recursive, cntx);
-            } else {
-              await component.run(steps, f, argv.recursive, cntx);
-            }
-          }
-        }
-      }
-    )
-    .argv;
+  .usage('Multi-component management system\nUsage:\n $0 <step[:step[...]]> [component[:component[:...]]] [options] -- [options]')
+  .help('help').alias('help', 'h')
+  .option('verbose', { alias: 'v', count: true, default: 0 })
+  .option('r', { describe: 'Execute commands recursively for all direct child components', alias: 'recursive', default: false, type: 'boolean' })
+  .option('p', { describe: 'Execute commands for multiple components in parallel', alias: 'parallel', default: false, type: 'boolean' })
+  .option('a', { describe: 'Show all components', alias: 'all', default: false, type: 'boolean' })
+  .option('u', { describe: 'Don\'t do anything, just print generated scripts', alias: 'dry-run', default: false, type: 'boolean' })
+  .option('e', { describe: 'Set environment variables', alias: 'env', default: [], type: 'array' })
+  .option('env-file', { describe: 'Read in a file of environment variables', default: [], type: 'array' })
+  .option('anchor-path', { describe: 'Shared components will be deployed using this path or project\'s root otherwise, if parameter is not defined', default: null })
+  .command(
+    /**************************************************************************/
+    'config', 'Create tln config in current folder, or clone/pull git repo with shared configuration',
+    (yargs) => {
+      yargs
+        .option('repo', { describe: 'Git repository url', alias:'repository', default: '', type: 'string' })
+        .option('f', { describe: 'Force override config file, if exists', alias: 'force', default: false, type: 'boolean' })
+        .option('q', { describe: 'Remove help information from the template', alias: 'quite', default: false, type: 'boolean' })
+    },
+    async (argv) => {
+      const a = appl.create(argv.verbose, process.cwd());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "a" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'a' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AUnusedVariable)